pyuvdata/utils.py

```diff
@@ -315,14 +315,15 @@ def fits_indexhdus(hdulist):
 def polstr2num(pol):
     """
     Convert polarization str to number according to AIPS Memo 117.
+    Use 'pI', 'pQ', 'pU' and 'pV' to make it clear that these are pseudo-Stokes, not true Stokes
 
     Args:
         pol: polarization string
 
     Returns:
         Number corresponding to string
     """
-    poldict = {'I': 1, 'Q': 2, 'U': 3, 'V': 4,
+    poldict = {'PI': 1, 'PQ': 2, 'PU': 3, 'PV': 4,
```

Comment:
this is supposed to be lower case on the `p`, no?

Reply:
ah, is this b/c you do `pol.upper()` before passing through to the dictionary?

Reply:
Yes, but the way the `polstr2num` function handles case insensitivity is to make them all upper case. I decided case-insensitivity was a good thing for `polstr2num`, but I used lowercase `p` for `polnum2str`.

Reply:
ok makes sense. i'm fine w/ this then.

Reply:
I can add a comment to that effect...